apiserver/worker/backend.py

```diff
@@ -3,14 +3,24 @@
 from hashlib import md5
 import json
 import os
-from time import gmtime, strftime
+from time import gmtime, strftime, sleep
+
+
+# Needs to match corresponding value in apiserver/config.py
+# Default value, 100 MiB
+MAX_BOT_UPLOAD_SIZE = 100 * 1024 * 1024
```

Comment:
At first I thought this was now redundant, but I see it's used as a default when the config doesn't define a value. The comment should probably be updated to reflect that.